app/(pages)/orders/view-order/_components/Navigator/Navigator.tsx

```diff
@@ -0,0 +1,83 @@
+'use client';
+
+import React from 'react';
+import Image from 'next/image';
+import { useState } from 'react';
+
+import styles from './Navigator.module.scss';
+
+export default function Navigator() {
+  const arrowSize = 18;
+  const [selected, setSelected] = useState(0);
+
+  function decrSelected() {
```

Comment:
same as comment below but negative numbers modulo are weird in js. If you want to make it the same, you can do:

(var - 1 + x) % x to ensure that whatever you're modding is positive first. Not sure why js doesn't just handle negative modulos normally